src/index.js

```diff
@@ -3,9 +3,4 @@ import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
 
-ReactDOM.render(
-  <React.StrictMode>
```

Comment:
Too restrictive or noisy?

Reply:
yeah noisy, mostly about dependencies that we have no control over